importer/views.py

```
@@ -319,6 +352,87 @@ def post(self, request):
+    except OSError as e:
+        logger.warning("OS error accessing %s: %s", path, e)
+
+    return entries
+
+
+class DirectoryListView(View):
+    """
+    API endpoint that returns directory contents as JSON.
+    """
+
+    def get(self, request):
```

Ruff (0.15.0)

[warning] Unused method argument: `request`

(ARG002)